content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -241,6 +305,35 @@ Packing external modules: @prisma/client@^5.1.1
 </cmdResult>
 </CodeWithResult>
 
-## Summary
+## SST guidance
+
+This section covers what to know about using Prisma in an SST app. If you're looking for notes for [AWS Lambda in general](#general-aws-lambda-guidance) or [Serverless Framework](#serverless-framework-guidance), see the other relevant sections above.
+
+### Working with environment variables
+
+While SST supports `.env` files, [it is not recommended](https://docs.sst.dev/config#should-i-use-configsecret-or-env-for-secrets). SST recommends using `Config` to access these environment variables in a secure way.
+
+The SST guide [available here](https://docs.sst.dev/config#overview) is a step-by-step guide to get started with Config. Assuming you have created a new secret called `DATABASE_URL` and have [bound that secret to your app](https://docs.sst.dev/config#bind-the-config) you can set up `PrismaClient` with the following:
+
+```ts file=prisma.ts
+import { PrismaClient } from '@prisma/client'
+import { Config } from 'sst/node/config'
 
-You have successfully deployed a Prisma-backed Serverless Framework application. Congratulations!
+// PrismaClient is attached to the `global` object in development to prevent
+// exhausting your database connection limit.
+//
+// Learn more:
+// https://pris.ly/d/help/next-js-best-practices
```

Comment:
Why link to the Next.js best practices?
Are we missing more general best practices?
What happens if you don't do that?

Reply:
Where is that advice from?

Reply:
I copy pasta'd this from our prisma examples repo. Oops.
```suggestion
```

Reply:
I actually thought that was Next.js specific advice.